types/Proposal0_3_0.ts

```diff
@@ -0,0 +1,35 @@
+import { MichelsonMap } from "@taquito/taquito";
```

Comment:
Why is it `0.3.0` and not `0.2.0`?

Reply:
The followings are different contract versions. the interfaces are break and don't compatible with previous one. `0.2.x` supports ticket but we didn't prioritize for UI to support it before.
- 0.1.x: current one used on mainnet.
- 0.2.x: base on 0.1.x and support the ticket 
- 0.3.x: base on 0.1.x and support TZIP-26